src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+
```

Comment:
Docstring to describe class and methods might make it clearer for future development?